src/mirakuru/base_env.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2014 by Clearcode <http://clearcode.cc>
```

Comment:
`# Copyright (C) 2015, 2016 by Clearcode <http://clearcode.cc>`

Or maybe just `2016`. Some projects list all years (since if not for Disney, the early years would show when copyright expires), other list the newest one only to keep it short.